src/mds/agg_mds/adapters.py

```diff
@@ -139,6 +139,9 @@ def mapFields(item: dict, mappings: dict, global_filters: list = []) -> dict:
         :return:
         """
 
+        if global_filters is None:
+            global_filters = []
```

Comment:
why this change?

Reply:
Noticed that it was being flagged (PyCharm flagged it as well as pylint) as a Mutable Default Argument. See https://docs.python-guide.org/writing/gotchas/ for a description and resolution.

Reply:
ah cool! thanks